circuits/lib/query/comparators.circom

```diff
@@ -21,7 +21,7 @@ template IN (valueArraySize){
         }
 
         // Greater than
-        component gt = GreaterThan(252);
+        component gt = GreaterThan(16);
```

Comment:
can you please explain why we init only with 16 bits

Reply:
Because it's for checking counter is greater than 0. Max counter value in our current setup is 64, which is 6 bits. Put there 16 just in case we would have very big value arrays.